api/v2alpha1/pipeline_spec.proto

```diff
@@ -1115,4 +1128,7 @@ message PipelineConfig {
   // Time to live configuration after the pipeline run is completed for
   // ephemeral resources created by the pipeline run.
   int32 resource_ttl = 3;
+
+  // Configuration for the workspace

```

Comment:
can we provide a bit more description, 1-2 sentences for what this is and what it's used for